services/upload/index.ts

```diff
@@ -0,0 +1,19 @@
+import { uploadToNftStorage } from './nft-storage'
+import { uploadToPinata } from './pinata'
+
+export const serviceType = {
```

Comment:
This can also be changed to 

```suggestion
export type UploadServiceType = 'nft-storage' | 'pinata'
```